### PR TITLE
Changes to Marketing Lead JD

### PR DESCRIPTION
**Pull request by carojacob**

### CODE REVIEW

File: _jobs/Marketing-Lead.md [hunk 1]
```diff
@@ -18,32 +18,32 @@ slug: ''
...
+    <li><p><b>Designing and executing an editorial plan for content marketing</b> that showcases OpenZeppelin’s offerings and benefits to our target audience. Identify topics and motivate the entire organization and external community members to create original content. This may include blog posts, research and reports, guides, case studies, presentations and sales material.</p></li>
+    <li><p><b>Defining an event attendance strategy</b> by sourcing, planning, and coordinating meetups, hackathons and industry events around the world. Find speaking opportunities at conferences and define actions to drive lead generation and increase brand awareness and community engagement as a result of them.</p></li>
+    <li><p><b>Coordinating product launches, new releases and company announcements</b>. Partner with Product, Open Source and Services teams to craft messaging and create material.Define and execute distribution strategy through various channels, including managing relationships with journalists and influencers, and working with a PR agency.</p></li>
```
Comment (elopio) [inline _jobs/Marketing-Lead.md:32]:
```suggestion
    <li><p><b>Coordinating product launches, new releases and company announcements</b>. Partner with Product, Open Source and Services teams to craft messaging and create material. Define and execute distribution strategy through various channels, including managing relationships with journalists and influencers, and working with a PR agency.</p></li>
```

File: _jobs/Marketing-Lead.md [hunk 2]
```diff
@@ -18,32 +18,32 @@ slug: ''
...
   <h3 class="job-description-title">You have</h3>
   <ul>
-    <li><p><b>3+ years of previous experience working on international marketing strategies in technology companies</b>, including specific experience with content creation, content marketing and event management to generate demand.</p></li>
-    <li><p><b>Superb organization and project management skills</b>. Ability to work on and manage multiple complex projects with multiple stakeholders, at once</p></li>
+    <li><p><b>3+ years of previous experience working on international marketing strategies in top technology or cybersecurity companies</b>. Specific experience with designing, executing, and tracking results of multiple lead generation programs including upsell & cross-sell campaigns, content marketing, email campaigns, and event management.</p></li>
+    <li><p><b>Experience working in a technical industry and work environment with a strong B2B component</b>. Experience in the cybersecurity space and understanding of developer audiences and blockchain technology are big pluses.</p></li>
+    <li><p><b>A startup mentality with a bias to action</b> and the ability to flex in a fast-paced environment.</p></li>
```
Comment (nventuro) [inline _jobs/Marketing-Lead.md:43]:
```suggestion
    <li><p><b>A startup mentality with a bias to action</b> and the ability to be flexible in a fast-paced environment.</p></li>
```
Likely this is what you meant:

flex: intransitive verb
 - flex one's muscles
 : to demonstrate one's strength
  __'an exaggerated need to flex his political muscles'__

(https://www.merriam-webster.com/dictionary/flex))

Comment (nventuro) [reply _jobs/Marketing-Lead.md:43]: Looks like I arrived 30 seconds late @elopio 😂

File: _jobs/Marketing-Lead.md [hunk 3]
```diff
@@ -18,32 +18,32 @@ slug: ''
...
+    <li><p><b>Superb organization and project management skills</b>. Ability to work on and manage multiple complex projects with multiple stakeholders, at once.</p></li>
     <li><p><b>Excellent written and verbal communication skills</b>. Proven ability to communicate complex topics with simplicity and clarity, and to communicate powerful stories to drive demand. Proven ability to create, edit and promote written and visual content.</p></li>
-    <li><p><b>Ability to work and adapt in a changing and technical industry and work environment</b>. Understanding of developer audiences, blockchain technology and the general state of the crypto industry are big pluses.</p></li>
-    <li><p><b>Ability to think critically</b> about complex problems with an analytical mindset, appropriately weight many factors and reach a definitive conclusion.</p></li>
-    <li><p><b>A startup mentality with a bias to action</b> and the ability to flex in a fast-paced environment</p></li>
+    <li><p><b>Ability to think critically about complex problems with an analytical mindset</b>,appropriately weigh many factors and reach a definitive conclusion.</p></li>
```
Comment (nventuro) [inline _jobs/Marketing-Lead.md:46]:
```suggestion
    <li><p><b>Ability to think critically about complex problems with an analytical mindset</b>, appropriately weigh many factors and reach a definitive conclusion.</p></li>
```

Missing space

**Comment by elopio** [comment]
Oh, sorry @nventuro. You caught more suggestion than in my review.

**Comment by elopio** [comment]
@nventuro @carojacob Let me make a couple of pull requests for Nico's suggestions.

**Comment by elopio** [comment]
pull requests for @nventuro's suggestions:
https://github.com/OpenZeppelin/openzeppelin.com/pull/97
https://github.com/OpenZeppelin/openzeppelin.com/pull/98

**Pull request merged**